[#171705742] Add Startup IDPS metadata setting procedure into README (#596)
pagopa/io-backend: README.md

$ kubectl create secret generic spid-cert --from-file=./cert.pem --from-file=./key.pem
```

### SAML (SPID) certs with ENV vars

The certificate and the secret key could be provided to the backend with two ENV variables:
- `SAML_KEY`
- `SAML_CERT`

if these variables are set `cert.pem` and `key.pem` are ignored.

### SPID Identity Providers Metadata

> When backend starts, SPID login is configured with Identity Providers metadata fetched from remote URL. URLs are provided by the following Environment variables: `CIE_METADATA_URL` (CIE login) `IDP_METADATA_URL` (SPID login) and `SPID_TESTENV_URL` (SPID test environment). Before that the backend become ready a time consuming operation must be completed (fetching remote metadata).

Reviewer: can you add a table with the default values from both test (local development) and prod environments ?

Author: Added into 344c47b

The default values are explained into the following table

| ENV                                    | stage                                                                             | production                                                       |
|----------------------------------------|-----------------------------------------------------------------------------------|-----------------------------------------------------------------|
| `IDP_METADATA_URL`                       | https://registry.spid.gov.it/metadata/idp/spid-entities-idps.xml                  | https://registry.spid.gov.it/metadata/idp/spid-entities-idps.xml |
| `CIE_METADATA_URL`                       | https://idserver.servizicie.interno.gov.it:8443/idp/shibboleth                    | https://idserver.servizicie.interno.gov.it/idp/shibboleth        |

> | `SPID_TESTENV_URL`                       | https://spid-testenv.dev.io.italia.it                                             | -                                                                |

Reviewer: it looks like we won't have a staging environment deployed (only production)
so these settings here should be the ones relative to local development setup (through docker)

Reviewer: can you add the settings to setup spid validator integration ? (local dev only, to let developers login using docker-compose and verify spid validator constraints)

Author: Fixed

The default values are explained into the following table

| ENV                                    | development                                                                 | production                                                       |
|----------------------------------------|-----------------------------------------------------------------------------|------------------------------------------------------------------|
| `IDP_METADATA_URL`                     | https://registry.spid.gov.it/metadata/idp/spid-entities-idps.xml            | https://registry.spid.gov.it/metadata/idp/spid-entities-idps.xml |
| `CIE_METADATA_URL`                     | https://idserver.servizicie.interno.gov.it:8443/idp/shibboleth              | https://idserver.servizicie.interno.gov.it/idp/shibboleth        |
| `SPID_TESTENV_URL`                     | https://spid-testenv2:8088                                                  | -                                                                |

> On local development only spid-testenv2, executed with docker-compose, could be used to complete a SPID login.

Reviewer: ```suggestion
For local development only, spid-testenv2 (executed with docker-compose) could be used to login with a (fake) SPID account.
```

> On production in order to reduce the startup time could be provided another Environment varible `STARTUP_IDPS_METADATA`. The command that must be used to set the variable is: 

Reviewer: ```suggestion
On production, in order to reduce the startup time, another Environment variable could be provided  `STARTUP_IDPS_METADATA`. To set its value run: 
```
